### PR TITLE
MULE-10556: Fix performance tests and run them in main build as junit…

### PR DESCRIPTION
**Pull request by LucianoGandini**
… tests so future errors are catched faster.

### CODE REVIEW

File: tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+package org.mule.test.performance.util;
+
+
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+
+import org.databene.contiperf.junit.ContiPerfRule;
+import org.junit.Rule;
+
+public class AbstractPerformanceTestCase extends AbstractMuleContextTestCase {
```
Comment (elrodro83) [inline tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java]: This class only has the rule. I think it's not necessary.

Comment (aiannucci) [inline tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java]: Although it defines an increased timeout. If you don't define this, many performance tests will fail

Comment (elrodro83) [inline tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java]: Also, configure the test timeout via system property passed through surefire (only when actually running the performance tests)

Comment (LucianoGandini) [inline tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java]: Created profile to pass the timeout and ContiPerf property only when runing perf tests.

Comment (LucianoGandini) [inline tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java]: Also the class is now the usual "AbstractMuleContextTestCase"

**Pull request merged**